hs_swat_modelinstance/templates/includes/application_scripts.html

```diff
@@ -0,0 +1,17 @@
+{% if debug %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.marionette.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-relational.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/moment-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/jqcsrf.js'></script>
+{% else %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.marionette.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-relational.min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/moment-min.js'></script>
```

Comment:
Is this backbone/marionette stuff actually even used anywhere? I don't see it being utilized in the template below.